packages/rich-text/src/create.js

```diff
@@ -297,11 +297,10 @@ function createFromElement( {
 
 	const length = element.childNodes.length;
 
-	// Remove any line breaks in text nodes. They are not content, but used to
-	// format the HTML. Line breaks in HTML are stored as BR elements.
-	// See https://www.w3.org/TR/html5/syntax.html#newlines.
 	const filterStringComplete = ( string ) => {
-		string = string.replace( /[\r\n]/g, '' );
+		// Reduce any whitespace used for HTML formatting to one space
+		// character, because it will also be displayed as such by the browser.
+		string = string.replace( /[\n\r\t]+/g, ' ' );
```

Comment:
Would it make sense to replace `\r` and `\n` with no spaces as it was before, and any number of tabs with one space?

Reply:
No, one of the issues this or fixes is exactly replacing line breaks with a space.